adafruit_character_lcd/character_lcd.py

```diff
@@ -84,7 +90,7 @@ def _set_bit(byte_value, position, val):
     return ret
 
 
-def _map(xval, in_min, in_max, out_min, out_max):
+def _map(xval: int, in_min: int, in_max: int, out_min: int, out_max: int) -> float:
```

Comment:
They only get used as `int` here, but for futureproofing, let's type all of these as float since there's no reason a function like this couldn't also be used for `float` values.